client/js/lounge.js

```diff
@@ -276,6 +276,13 @@ $(function() {
 		return msg;
 	}
 
+	function pushHistoryState(state) {
```

Comment:
This is small enough to be inclined, imo. Then you can avoid handling the click code completely if history is not supported.